src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -497,13 +402,13 @@ internal byte[] GetOutgoingBlob(byte[] incomingBlob, bool throwOnError, out Inte
             }
 
 
-            if (((int)statusCode & unchecked((int)0x80000000)) != 0)
+            if (NegoState.IsError(statusCode))
             {
                 CloseContext();
                 _isCompleted = true;
                 if (throwOnError)
                 {
-                    var exception = new Win32Exception((int)statusCode);
+                    Exception exception = NegotiateStreamPal.CreateExceptionFromError(statusCode);
```

Comment:
I presume that now there is no reason for this type change here? The PAL will always return a `Win32Exception` on all platforms.